### ISSUE TITLE
ZPL to image?

### ISSUE THREAD

**Issue by mehov** [metafloor/zpl-image#8]
Hey,

I'm sorry if I'm missing something obvious, but is it possible to convert ZPL to an image or a PDF using this library? Basically into anything that can be embedded into a browser page. 

Thank you!

**Comment by metafloor** [comment]
No, this library just converts image formats (png, jpg, gif, etc...) to the ZPL GRF bitmap format.  (ZPL's native image format.)

To create an image of a label, take a look at labelary.com.  It provides a publicly accessible API for rendering ZPL label formats to either image or PDF.

**Comment by mehov** [comment]
Thank you for a quick reply.

Labelary won't work mainly because it's a 3rd party external solution (reliability and dependency risks, privacy risks, licensing fees).

If someone finds this on Google search: currently looking into https://github.com/MatthAmoros/shard-zpl-viewer.  Will update the answer if I find more.
